### DISCUSSION TITLE
Problem hosting this with custom jitsi instance

### DISCUSSION THREAD

**Discussion by LennyPenny** [Chatmosphere/chatmosphere-app#18] [Q&A]
Hey, 
we really love this project and would love to host this for ourselves. 

We installed the application and edited the `serverConfig.ts` to use our custom jitsi instance. It now looks like this: 
```ts
export const connectionOptions = {
  hosts: {
     domain: 'meet.flipdot.dev',
     muc: 'conference.meet.flipdot.dev', 
  }, 
  bosh: `//meet.flipdot.dev/http-bind`,
  clientNode: 'http://jitsi.org/jitsimeet', 
}

```

It all starts fine but when we join a session we are always alone in the room, even though multiple people are on the same chatmosphere session. 

Is there any special config you have to set on the jitsi instance to make this work?

**Comment by n-hallberg** [comment]
Hi :)
Got the same problem. I tried adding this:

```
enableP2P: true,
  p2p: {
    enabled: true,
    preferH264: true,
    disableH264: true,
    useStunTurn: true,
  },
```
to the serverConfig.ts, but the problem persists.
  
Would be great to find out how it works!  
Its really an awesome project!

**Comment by dkgrieshammer** [comment]
This sounds like a misconfig in your Jitis. How did you setup your jitsi Instance? Is Chatmosphere on the same Instance or Cross Origin (thats what we do)? For a first hint maybe have a look at "shit its not working with more than 2 people" here
https://www.notion.so/Server-Setup-932469ca0423488fb8ee655954a4f121
